internal/backends/collection_test.go

```diff
@@ -189,19 +189,20 @@ func TestCollectionStats(t *testing.T) {
 				db, err := b.Database(dbName)
 				require.NoError(t, err)
 
+				var c backends.Collection
 				cNames := []string{"collectionOne", "collectionTwo"}
 				for _, cName := range cNames {
 					err = db.CreateCollection(ctx, &backends.CreateCollectionParams{Name: cName})
 					require.NoError(t, err)
-				}
 
-				c, err := db.Collection(cNames[0])
-				require.NoError(t, err)
+					c, err = db.Collection(cName)
+					require.NoError(t, err)
 
-				_, err = c.InsertAll(ctx, &backends.InsertAllParams{
-					Docs: []*types.Document{must.NotFail(types.NewDocument("_id", types.NewObjectID()))},
-				})
-				require.NoError(t, err)
+					_, err = c.InsertAll(ctx, &backends.InsertAllParams{
+						Docs: []*types.Document{must.NotFail(types.NewDocument("_id", types.NewObjectID()))},
+					})
```

Comment:
Insert in each collection so that L215 passes and ensures that size of one collection `res.SizeCollection` is smaller than the size of all collections in `dbStatsRes.SizeCollections`.